tests/test_send_plot.py

```diff
@@ -44,7 +46,9 @@ def test_send_plot():
 
     agentNetwork.bind_agents(gen_agent, monitor_agent)
 
-    gen_agent.dummy_send_graph()
+    gen_agent.dummy_send_graph(mode="image")
+    time.sleep(3)
+    gen_agent.dummy_send_graph(mode="plotly")
```

Comment:
I suggest, separating these two tests, so if one fails the other could still pass and in this case to even easier see, which one caused the failure.